fdbserver/ProxyCommitData.actor.h

```diff
@@ -225,6 +225,18 @@ struct ProxyCommitData {
 	// Each storage server's own team.
 	std::unordered_map<UID, ptxn::StorageServerStorageTeams> ssToStorageTeam;
 
+	// FIXME: This function should be replaced/removed when we do not do broadcasting
+	std::set<ptxn::StorageTeamID> getAllStorageTeamIDs() const {
```

Comment:
This is not used. We get all the teams for a tlog group from tLogGroupCollection.